### DISCUSSION TITLE
BTP Resource consumption monitoring configuration without application Log monitoring service

### DISCUSSION THREAD

**Discussion by Gagan-Zoetis** [SAP-samples/btp-resource-consumption-monitor#17] [Q&A]
Hello Team,

After implementing the application log service, we noticed that we are getting heavily charged for Application Log monitoring service on daily basis and for monthly basis it will be in thousands USD. Can we configure the BTP resource consumption monitoring without application log monitoring service and how we can reduce the billing?

Regards
Gagan

**Comment by willempardaens** [comment]
answered in #16
